test/helpers/formRenderer.spec.js

```diff
@@ -4,7 +4,8 @@ import sinon from 'sinon';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { expect } from 'chai';
-import ScriptRunner from '../../src/helpers/scriptRunner';
+import ScriptRunner from '../../src/helpers/ScriptRunner';
```

Comment:
Why have u chnaged this to captial S?? "ScriptRunner"?

Reply:
Normally we changed, i can revert it back to small s.

Reply:
Yes, it will break on other OS, as all OS is not case sensitive

Reply:
okay, added commit changing them